src/app/(post-login)/submission/program/[shortName]/clinical-submission/page.tsx

```diff
@@ -381,6 +378,37 @@ const ClinicalSubmission = ({ shortName }: { shortName: string }) => {
 							onSignOffClick={handleSignOff}
 							clinicalTypes={clinicalEntities.map(({ clinicalType }) => clinicalType)}
 						/>
+						{isPendingApproval && (
+							<div
+								css={css`
+									padding: 24px;
+								`}
+							>
```

Comment:
this is nested pretty deep, it could maybe benefit from moving some code into functions.